addons/binding/org.openhab.binding.helios/src/main/java/org/openhab/binding/helios/HeliosBindingConstants.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
```

Comment:
could you pls rerun "mvn license:format", so that the headers are updated to 2018?